README.md

```diff
@@ -10,52 +10,94 @@
+- [📣 What does this dbt package do?](https://github.com/fivetran/dbt_quickbooks_source/#-what-does-this-dbt-package-do)
+- [🎯 How do I use the dbt package?](https://github.com/fivetran/dbt_quickbooks_source/#-how-do-i-use-the-dbt-package) 
+    - [Required steps](https://github.com/fivetran/dbt_quickbooks_source/#step-1-prerequisites)
+    - [Additional options](https://github.com/fivetran/dbt_quickbooks_source/#optional-step-5-orchestrate-your-models-with-fivetran-transformations-for-dbt-core)
+- [🔍 Does this package have dependencies?](https://github.com/fivetran/dbt_quickbooks_source/#-does-this-package-have-dependencies)
+- [🙌 How is this package maintained and can I contribute?](https://github.com/fivetran/dbt_quickbooks_source/#-how-is-this-package-maintained-and-can-i-contribute)
+    - [Package Maintenance](https://github.com/fivetran/dbt_quickbooks_source/#package-maintenance)
+    - [Contributions](https://github.com/fivetran/dbt_quickbooks_source/#contributions)
+    - [🏪 Are there any resources available?](https://github.com/fivetran/dbt_quickbooks_source/#-are-there-any-resources-available)
+    
+# 📣 What does this dbt package do?
+- Materializes [QuickBooks staging tables](https://fivetran.github.io/dbt_quickbooks_source/#!/overview/quickbooks_source/models/?g_v=1&g_e=seeds) which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/quickbooks#schemainformation). These staging tables clean, test, and prepare your QuickBooks data from
+from [Fivetran's connector](https://fivetran.com/docs/applications/quickbooks) for analysis by doing the following:
+- Name columns for consistency across all packages and for easier analysis.
+- Adds descriptions to tables and columns that are synced using Fivetran
+- Models staging tables, which will be used in our transform package.
+- Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
+- Generates a comprehensive data dictionary of your source and modeled QuickBooks data through the [dbt docs site](https://fivetran.github.io/dbt_quickbooks_source/).
+- These tables are designed to work simultaneously with our [QuickBooks transformation package](https://github.com/fivetran/dbt_quickbooks/)
+
+# 🎯 How do I use the dbt package?
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
+
+- At least one Fivetran QuickBooks connector syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+
+## Step 2: Install the package
+Include the following QuickBooks package version in your `packages.yml` file.
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 
 ```yaml
 packages:
   - package: fivetran/quickbooks_source
     version: [">=0.6.0", "<0.7.0"]
```

Comment:
These changes should be breaking and we should increment the version here

Reply:
Incremented.